typed_python/types_test.py

```diff
@@ -171,6 +171,18 @@ def pickRandomly(self):
 
 class NativeTypesTests(unittest.TestCase):
 
+    def test_refcount_bug_with_simple_string(self):
+
+        with self.assertRaises(TypeError) as context:
```

Comment:
there is something called `assertRaisesRegex` that lets you simultaneously check the type of exception and also checks that a regex matches the internals.